edx/analytics/tasks/reports/reconcile.py

```diff
@@ -146,7 +147,11 @@ def requires(self):
             OrderTableTask(
                 import_date=self.import_date
             ),
-            PaymentTask(
+            # Temporarily disable this task for testing purposes:
+            # PaymentTask(
```

Comment:
@itsjeyd should not be commented - consider adding a new temporary branch with this change (or figure out what needs to be done to not comment this task)